src/domain/synthetics/tokens/useOracleKeeperFetcher.ts

```diff
@@ -127,11 +136,22 @@ export function useOracleKeeperFetcher(chainId: number) {
         });
     }
 
+    async function fetchIncentivesRewards(): Promise<RawIncentivesStats> {
+      return fetch(
+        buildUrl(oracleKeeperUrl!, "/incentives/stip/lp", {
+          ignoreStartDate: forceIncentivesActive ? "1" : undefined,
+        })
+      )
+        .then((res) => res.json())
+        .catch(() => null);
```

Comment:
does it make sense to log error? we do this for other requests

Reply:
we for some reason entirely ignore subgraph error here which I postpone to change. Probably needs to be updated here as well

Reply:
not sure i got it, this is a request to our API, not subgraph

Reply:
I mean we have two calls:
```
[
   client!.query({ query: gql(`{${queryBody}}`), fetchPolicy: "no-cache" }),
   shouldCalcBonusApr ? oracleKeeperFetcher.fetchIncentivesRewards() : Promise.resolve(null),
]
```
Both of them now fails silently